src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -193,8 +190,8 @@ private void ShowGoldBar()
 
             _infoBarOpen = true;
 
-            Debug.Assert(_experimentationService.IsExperimentEnabled(InternalFlightName) ||
-                         _experimentationService.IsExperimentEnabled(ExternalFlightName));
+            Debug.Assert(await _experimentationService.IsExperimentEnabledAsync(InternalFlightName, cancellationToken) ||
+                         await _experimentationService.IsExperimentEnabledAsync(ExternalFlightName, cancellationToken));
```

Comment:
the fact that the two boolean expressions that check these things in the file don't follow the same format bug me :)